src/diffusers/models/attention_processor.py

```diff
@@ -219,6 +219,7 @@ def __init__(
             self.to_v = None
 
         if self.added_kv_proj_dim is not None:
+            self.added_proj_bias = added_proj_bias
```

Comment:
ok but I think it's easier if you move this outside of the if, along with other attributes - that way `Attention` will always has this attribute